src/components/Stepper/Steps.tsx

```diff
@@ -42,7 +33,7 @@ const Steps = () => {
           <Stepper
             alternativeLabel={true}
             nonLinear={true}
-            activeStep={activeStep}
+            activeStep={(activeStep && activeStep.number) || undefined}
```

Comment:
You shouldn't assign `undefined`. Undefined means that is was never defined, please use `null`

Reply:
Stepper component from material ui is accepting number or undefined . What in this case?

Reply:
Well then use undefined :)

Reply:
Or the first one as the default?